### PR TITLE
Use hub.arenadata.io/adcm/adcm as ADCM images source

### PR DESCRIPTION
**Pull request by dgusakov**
Use hub.arenadata.io/adcm/adcm as ADCM images source

Fix typos in docs

### CODE REVIEW

**Review by vgorkavenko — DISMISSED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Review by madlabman — APPROVED** [review]

File: src/adcm_pytest_plugin/docker_utils.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def run_adcm(
         If tag is None or is not present than a tag 'latest' will be used
         """
         if image is None:
```
Comment (madlabman) [inline src/adcm_pytest_plugin/docker_utils.py:256]: `if not image`? :]

**Review by madlabman — APPROVED** [review]

**Review by Maunty — DISMISSED** [review]

File: src/adcm_pytest_plugin/plugin.py [hunk 2]
```diff
@@ -135,12 +135,12 @@ def parametrized_by_adcm_version(adcm_min_version=None, adcm_images=None):
...
 
 def _get_adcm_new_versions_tags(min_ver):
-    response = requests.get(
-        "https://hub.docker.com/v2/repositories/arenadata/adcm/tags/?page_size=100"
-    )
-    for data in response.json()["results"]:
-        tag = data.get("name")  # get tag
-        if tag.isdigit():  # filter latest tag
+
+    tags = requests.get("https://hub.arenadata.io/v2/adcm/adcm/tags/list").json()[
```
[the remark this answers is not in the document]
Comment (dgusakov) [reply src/adcm_pytest_plugin/plugin.py:139]: This is the only place where we use it. No need for it now

File: src/adcm_pytest_plugin/docker_utils.py (hunk 1 above)
Comment (dgusakov) [reply src/adcm_pytest_plugin/docker_utils.py]: Done

File: src/adcm_pytest_plugin/plugin.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Maunty) [reply src/adcm_pytest_plugin/plugin.py:139]: More over, response in not fully compatible with rocket hub one. So there is no reason to make url as global variable.

[the remark this answers is not in the document]
Comment (madlabman) [reply src/adcm_pytest_plugin/plugin.py:139]: Actually I've noticed for a better formatting :)

**Review by vgorkavenko — APPROVED** [review]

**Review by madlabman — APPROVED** [review]

**Pull request merged**